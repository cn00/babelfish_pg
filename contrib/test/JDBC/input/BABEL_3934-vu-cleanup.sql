DROP PROCEDURE babel_3934_1 --should fail, procedure should not be created
GO

DROP PROCEDURE babel_3934_2
GO