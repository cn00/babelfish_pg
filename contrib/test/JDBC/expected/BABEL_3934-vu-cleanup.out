DROP PROCEDURE babel_3934_1 --should fail, procedure should not be created
GO
~~ERROR (Code: 3701)~~

~~ERROR (Message: could not find a procedure named "babel_3934_1")~~


DROP PROCEDURE babel_3934_2
GO
